backend/test/chat.e2e-spec.ts

```diff
@@ -5,10 +5,21 @@ import { ChatController } from '../src/chats/chat/chat.controller';
 import * as request from 'supertest';
 import { Chat } from '../src/chats/chat/entities/chat.entity';
 import { AllTestingModule } from '../src/shared/test.module';
+import {CreateMessageDto} from "../src/chats/message/dto/create-message.dto";
+import {MessageService} from "../src/chats/message/message.service";
+import {MessageController} from "../src/chats/message/message.controller";
 
 describe('chat e2e', () => {
 	let app: INestApplication;
 	let chatController: ChatController;
+	let messageService: MessageService;
+	let messageController: MessageController;
+
+	const testMessages: CreateMessageDto[] = [
+		{ sender_id: 1, chat_id: 1, content: 'hello' },
+		{ sender_id: 2, chat_id: 1, content: 'world!' },
+	];
```

Comment:
This is added but not used in the e2e test

(same for MockChat a few lines below this)